docs/amd_sev_snp.md

```diff
@@ -0,0 +1,38 @@
+# AMD SEV-SNP
+
+### WARNING
+This feature is only supported on MSHV and not KVM for the time being.
+
+AMD Secure Encrypted Virtualization & Secure Nested Paging (SEV-SNP) is an AMD
+technology designed to add strong memory integrity protection to help prevent
+malicious hypervisor-based attacks like data replay, memory-remapping and more
+in order to create an isolated execution environment. Here are some useful
+links:
+
+* [SNP Homepage] (https://www.amd.com/en/processors/amd-secure-encrypted-virtualization)
+more information about SEV-SNP technical aspects, design and specification.
+
+## Cloud Hypervisor support
+
+It is required to use a machine which has enabled support for AMD SEV-SNP in
+the BIOS.
+
+On the Cloud Hypervisor side, all you need is to build the project with the 
+`sev_snp` feature enabled:
+
+```bash
+cargo build --features "mshv,sev_snp"
```

Comment:
You will need `--no-default-features`, otherwise the `kvm` feature will be enabled by default.

Reply:
Done